core-implementation/record_transaction/record_transaction.test.ts

```diff
@@ -29,4 +29,4 @@ test("record a transaction", (done) => {
 			expect(response?.succeeded).toEqual(true);
 		},
 	);
-});
```

Comment:
please setup prettier in your ide to avoid these type of changes

i have mine setup to "format on save", so whenever i press ctrl+s, it formats the entire file automatically.